README.md

```diff
@@ -1,26 +1,57 @@
-# Offscreen Effect Player module
+# [Offscreen Effect Player module (OEP-module)](https://docs.banuba.com/face-ar-sdk-v1/core/cpp_api_overview#offscreen-effect-player)
+
+[Description of the **Offscreen Effect Player** on the Banuba SDK website](https://docs.banuba.com/face-ar-sdk-v1/core/cpp_api_overview#offscreen-effect-player)
 
 ## Description
 
-Common submodule for some projects using **offscreen_effect_player**
-The module implements interaction with the effect player. The module consists of six main interfaces:
+Common submodule for some projects using [**Offscreen effect player**](./interfaces/offscreen_effect_player.hpp). The module implements interaction with the effect player.
+
+## Folders structure
+
+- [**docs**](./docs/) - illustrations used in this README.md
+- [**interfaces**](./interfaces/) - offscreen effect player interfaces
+- [**offscreen_effect_player**](./offscreen_effect_player/) - is a wrapper for effect_player. It allows you to use your own implementation for offscreen_render_target
```

Comment:
Vague...
- [**offscreen_effect_player**](./offscreen_effect_player/) - contains the implementation of INTERFACE. The implementation manages rendering via ofscreen_render_target interface and manages effect_player...

Something like that - write what it does, instead of stating something unrelated.

Reply:
The same for other pieces of this section